app/channel_test.go

```diff
@@ -452,7 +452,9 @@ func TestAddChannelMemberNoUserRequestor(t *testing.T) {
 	}
 	assert.Equal(t, groupUserIds, channelMemberHistoryUserIds)
 
-	postList := store.Must(th.App.Srv.Store.Post().GetPosts(channel.Id, 0, 1, false)).(*model.PostList)
+	postList, err := th.App.Srv.Store.Post().GetPosts(channel.Id, 0, 1, false)
+	require.Nil(t,err)
```

Comment:
A small note that `require.NoError()` will also work for times like this. It reads a little better when viewing the test.